docs/configurations/index.rst

```diff
@@ -8,6 +8,15 @@ and how to create new ones.
 
 OCIO Configurations can be downloaded here: `.zip <http://github.com/imageworks/OpenColorIO-Configs/zipball/master>`__ `.tar.gz <http://github.com/imageworks/OpenColorIO-Configs/tarball/master>`__ (OCIO v1.0+)
 
+OCIO Configurations for ACES 1.1 and 1.2 can be downloaded from the `Colour Science <https://github.com/colour-science/OpenColorIO-Configs>`_ repository maintained by Thomas Mansencal here:
```

Comment:
`Colour Science` -> `Colour Science OpenColorIO-Configs`

`repository` -> `fork`

Reply:
Just for clairty

Reply:
and `maintained by Thomas Mansencal here:` --> `maintained by Thomas Mansencal and Michael Parsons here:`

Technically @mlfp is also an official maintainer!

Reply:
Changes made, just need to push. @KelSolaar I also put that into the README :+1: